modules/local/mtx_to_h5ad.nf

```diff
@@ -0,0 +1,51 @@
+process MTX_TO_H5AD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "conda-forge::scanpy conda-forge::python-igraph conda-forge::leidenalg" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/scanpy:1.7.2--pyhdfd78af_0' :
+        'quay.io/biocontainers/scanpy:1.7.2--pyhdfd78af_0' }"
+
+    input:
+    // inputs from cellranger nf-core module does not come in a single sample dir
+    // for each sample, the sub-folders and files come directly in array.
+    tuple val(meta), path(inputs)
+
+    output:
+    path "*.h5ad", emit: h5ad
+
+    script:
+    if (params.aligner == 'cellranger')
+    """
+    # convert file types
+    cellranger_mtx_to_h5ad.py \\
+        -m filtered_feature_bc_matrix \\
+        -o matrix.h5ad
```

Comment:
why have different h5ad filenames for each aligner?

Reply:
Hi @grst,
It turns out that to have a concat module (files from different samples in same workdir) filenames cannot be equal, so I changed the matrix to be named `${meta.id}_matrix.h5ad` for all aligners.